sdk/servicebus/azure-servicebus/tests/stress_tests/test_stress_queues.py

```diff
@@ -10,7 +10,7 @@
 import sys
 import time
 
-from azure.servicebus import ServiceBusClient
+from azure.servicebus import ServiceBusClient, AutoLockRenew
```

Comment:
```suggestion
from azure.servicebus import ServiceBusClient, AutoLockRenewer
```

Reply:
Heh, I think these two prs passed each other in flight, so thank you for the callout.